src/test/java/com/jcabi/aspects/CacheableTest.java

```diff
@@ -53,8 +54,11 @@
     /**
      * Cacheable can cache calls.
      * @throws Exception If something goes wrong
+     * @todo Make sure that this test runs under Java 8 and remove the Ignore
```

Comment:
Puzzle format here is wrong. Should be `@todo #<current issue number>:<estimate>`. E.g. `@todo #126:30min`.